src/Tizen.NUI/src/public/WebView/WebView.cs

```diff
@@ -483,6 +486,29 @@ protected override void ReleaseSwigCPtr(System.Runtime.InteropServices.HandleRef
             }
         }
 
+        /// <summary>
+        /// Event for the NavigationPolicyDecided signal which can be used to subscribe or unsubscribe the event handler.<br />
+        /// This signal is emitted when response policy would be decided.<br />
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public event EventHandler<WebViewPolicyDecidedEventArgs> NavigationPolicyDecided
+        {
+            add
+            {
+                if (navigationPolicyDecidedEventHandler == null)
+                {
+                    navigationPolicyDecidedCallback = OnNavigationPolicyDecided;
+                    IntPtr ip = Marshal.GetFunctionPointerForDelegate(navigationPolicyDecidedCallback);
+                    Interop.WebView.RegisterNavigationPolicyDecidedCallback(SwigCPtr, new HandleRef(this, ip));
+                }
+                navigationPolicyDecidedEventHandler += value;
+            }
+            remove
```

Comment:
this looks different from other `event` code. 
could you check this for example?
(ex) https://github.com/Samsung/TizenFX/blob/master/src/Tizen.NUI/src/public/BaseComponents/ViewEvent.cs#L127

it seems that only register is existed. (but is unregistered code needed ?)

Reply:
c# callbacks could not be called again after xxx handler is removed.
so there is no severe issue now.
If code for unregsiterxxxcallback must be added, some code in csharp binder need be changed.
I will fix them in another patches.